yt/yt/server/node/data_node/session_detail.cpp

```diff
@@ -34,7 +34,7 @@ TSessionBase::TSessionBase(
     , Location_(location)
     , Lease_(std::move(lease))
     , MasterEpoch_(Bootstrap_->GetMasterEpoch())
-    , SessionInvoker_(CreateSerializedInvoker(Location_->GetAuxPoolInvoker()))
+    , SessionInvoker_(CreateSerializedInvoker(Location_->GetAuxPoolInvoker(options.WorkloadDescriptor, SessionId_.ChunkId)))
```

Comment:
It uses the implicit knowledge that the session Id is a chunk Id

Reply:
Yes. You suggest adding `.ToString()` to `SessionId_`?